### ISSUE TITLE
environments dir placement

### ISSUE THREAD

**Issue by galki** [nstudio/xplat#49]
For the sake of consistency (with Angular), can we have the `environments` dir on the same level as `libs`, `apps` and `xplat` instead of inside `libs/core`?

**Comment by NathanWalker** [comment]
Hi @galki definitely with ya here. I've debated this a bit internally. So Nx actually adds `environments` directory to each web app.

We could have one parallel to the other main folders however that would mean more tsconfig modifications to reference it. 

Any particular reason you have trouble with it being in `libs/core`? That location makes environment data accessible to any app in workspace, even cross platform apps.

**Comment by galki** [comment]
Right, that makes sense. Perhaps its just a naming problem. Now that I think about it the below feels natural too:

- core
  - base
  - environments
  - services
  - features
    - ui
  - scss
  - utils
- xplat

`libs` becomes `core` as its the "core" of the project. Importing would be simplified too.
